src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ListView/HorizontalListView_Templates.xaml

```diff
@@ -9,7 +9,7 @@
 	xmlns:ios="http://uno.ui/ios"
 	xmlns:uConverters="using:Uno.UI.Samples.Converters"
 	xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	xmlns:android="http://uno.ui/android"
+	xmlns:android="http://uno.ui/android#using:Uno.UI.Controls.Legacy"
```

Comment:
We need to document this in https://github.com/unoplatform/uno/blob/master/doc/articles/platform-specific-xaml.md